### DISCUSSION TITLE
Support updating and displaying `numeric` data type precision and scale within Table Editor UI

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#20383] [Feature Requests]
## Background
Postgres supports [arbitrary precision numbers](https://www.postgresql.org/docs/current/datatype-numeric.html), which lets you store decimal numbers at a specific precision and scale. This is useful for storing certain types of values (currency, ratings, etc.)

Currently, if a Supabase user wants to set the precision and scale for a column that is a `numeric` type, they can only do so via the SQL Editor through the following query:

```sql
alter table table_name
alter column column_name
set data type numeric(4,2)
```

This will enforce only supporting values from -99.99 to 99.99.

However, even when this query succeeds in the SQL Editor, there is nothing in the Table Editor displaying this constraint.

## Request

Support adding, removing, updating, and displaying the `numeric` precision and scale from the Table Editor.

**Comment by GameboyVlol** [comment]
pleaaaaaaaaaaaaaaaaaaaaase do this
